user_sync/config.py

```diff
@@ -129,7 +126,7 @@ def get_dashboard_options_for_accessors(self):
             accessor_config = None
             if (accessors_config != None): 
                 accessor_config = accessors_config.get_list(organization_name, True) 
-            accessor_config_sources = self.as_list(accessor_config)
+            accessor_config_sources = ConfigFileLoader.get_relative_filenames(self.as_list(accessor_config))
```

Comment:
I don't think this is right; it presumes that all of the configs are filenames, but of course they may not always be filenames.  This should move down, see below.